modules/apps/object/object-js-components-web/src/main/resources/META-INF/resources/tests/MultipleSelect.spec.tsx

```diff
@@ -0,0 +1,311 @@
+								checked: true,
+								label: 'Account Member',
+								value: 'Account Member',
+							},
+						],
+						label: 'Account Roles',
+						value: 'accountRoles',
+					},
+					{
+						children: [
+							{
+								_key: 'Administrator',
+								checked: true,
+								label: 'Administrator',
+								value: 'Administrator',
+							},
+							{
+								_key: 'Analytics Administrator',
+								checked: true,
+								label: 'Analytics Administrator',
+								value: 'Analytics Administrator',
+							},
+						],
+						label: 'Regular Roles',
+						value: 'regularRoles',
+					},
+					{
+						children: [
+							{
+								_key: 'Account Manager',
+								checked: true,
+								label: 'Account Manager',
+								value: 'Account Manager',
+							},
+							{
+								_key: 'Organization Administrator',
+								checked: true,
+								label: 'Organization Administrator',
+								value: 'Organization Administrator',
+							},
+						],
+						label: 'Organization Roles',
+						value: 'organizationRoles',
+					},
+				])
+			}
+		/>
+	);
+
+	fireEvent.click(screen.getByRole('combobox'));
+
+	fireEvent.click(screen.getByLabelText('select-all'));
+
+	expect(screen.getByLabelText('select-all')).toBeChecked();
+});
+
+it('check an option', () => {
+	render(
+		<MultipleSelect
+			disabled={false}
+			id="multipleSelect"
+			options={mockOptions}
+			selectAllOption
+			setOptions={(options) =>
+				expect(options).toStrictEqual([
+					{
+						children: [
+							{
+								_key: 'Account Administrator',
+								checked: true,
+								label: 'Account Administrator',
+								value: 'Account Administrator',
+							},
+							{
+								_key: 'Account Member',
+								checked: false,
+								label: 'Account Member',
+								value: 'Account Member',
+							},
+						],
+						label: 'Account Roles',
+						value: 'accountRoles',
+					},
+					{
+						children: [
+							{
+								_key: 'Administrator',
+								checked: true,
+								label: 'Administrator',
+								value: 'Administrator',
+							},
+							{
+								_key: 'Analytics Administrator',
+								checked: true,
+								label: 'Analytics Administrator',
+								value: 'Analytics Administrator',
+							},
+						],
+						label: 'Regular Roles',
+						value: 'regularRoles',
+					},
+					{
+						children: [
+							{
+								_key: 'Account Manager',
+								checked: false,
+								label: 'Account Manager',
+								value: 'Account Manager',
+							},
+							{
+								_key: 'Organization Administrator',
+								checked: true,
+								label: 'Organization Administrator',
+								value: 'Organization Administrator',
+							},
+						],
+						label: 'Organization Roles',
+						value: 'organizationRoles',
+					},
+				])
+			}
+		/>
+	);
+
+	fireEvent.click(screen.getByRole('combobox'));
+
+	fireEvent.click(screen.getByText('Account Administrator'));
+
+	expect(screen.getByLabelText('select-all')).not.toBeChecked();
```

Comment:
the title of the test is `"check an option"` but here you are only checking if the `select-all` is not checked. what is the real purpose of this test? if it is to check that only one option is checked, i think you should check if the option itself is checked as well. 

also the title `"check an option"` is not very clear, you could describe more what the test is about. if you put `"check only an option"` it already makes more clear of its objective.

Reply:
> the title of the test is "check an option" but here you are only verifying if the select-all is not checked.

Hello Rebeca, in fact the test is performing a check on the data structure returned by `setOptions`, using `expect().toStrictEqual()` in `setOptions` function.

Some multiple select actions are internal to clay and other actions are implemented on our side. In case we select an item, we are executing this [onChange](https://github.com/liferay/liferay-portal/blob/4da7307aa24e47b212e231802018cd7ced24bd70/modules/apps/object/object-js-components-web/src/main/resources/META-INF/resources/components/Select/MultipleSelect.tsx#L215-L246) of Multiple select, and this test aims to perform what we are returning to setOptions and setSelectAllChecked when we select an option.

Reply:
The main objective of the test is to verify that the recently clicked option, `'Account Administrator'`, is being returned correctly as a checked item.
```
{
  _key: 'Account Administrator',
  checked: true,
  label: 'Account Administrator',
  value: 'Account Administrator',
},
```
Note that this option is not marked in the mocked data structure.

Do you believe that I should still rename the test, emphasizing that only one option is being selected?

Reply:
i see, thanks for the explanation! in this case feel free to rename it or not :blush: